src/components/static/Title.jsx

```diff
@@ -18,13 +22,16 @@ const Title = () => {
           </div>
           <div className="text-lg lg:text-xl font-paragraph text-bear-teal-100">
             {CONFIG.date.toLocaleDateString(undefined, {
-              year: "numeric",
               month: "long",
               day: "numeric",
             })}
+            {"-"}
+            {CONFIG.lastDate.toLocaleDateString(undefined, { day: "numeric" })}
```

Comment:
shouldnt be undefined

Reply:
It was undefined so it defaults to the user's locale while an option can be passed in to select day/month/year (how it was done previously for a single date), but that doesn't really make sense anymore since individual values are being extracted. I'll rewrite that.